docs/buildah-bud.md

```diff
@@ -278,6 +278,12 @@ container technology).
 Note: You can also override the default isolation type by setting the
 BUILDAH\_ISOLATION environment variable.  `export BUILDAH_ISOLATION=oci`
 
+**--jobs** *N*
+
+Run up to N concurrent stages in parallel.  If the number of jobs is greater than 1,
```

Comment:
What is the default value?  1?

Reply:
What does value 0  Mean?

Reply:
the default value is 1.  0 is treated as 1 in the current implementation, should we raise an error instead?  make at least complains with `-j 0`:
```
$ make -j0 
make: the '-j' option requires a positive integer argument
```

Reply:
Why not make 0 unlimited and >1 be the number of threads, and eliminate the negative numbers?

Reply:
good idea.  Changed to accept only >= 0, with 0 unlimited